packages/UI/src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,527 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Button } from 'components/atoms';
+import { Checkbox } from './../Checkbox';
+import Modal, { Close, Description } from './Modal';
+
+/**
+ * @description A modal is a dialog box/popup window that is displayed on top of the current page
+ * https://www.radix-ui.com/primitives/docs/components/dialog
+ * Close export is an wrapper to close the modal from a trigger element.
+ * Description export is an wrapper to add a description to the modal for accessibility.
+ * z level is defaulted to z-50 to ensure it is on top of all other elements. This can be overridden by passing a className prop.
+ */
+
+const meta: Meta<typeof Modal> = {
+  component: Modal,
+};
+
+export default meta;
+type Story = StoryObj<typeof Modal>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/react/api/csf
+ * to learn how to use render functions.
+ */
+
+export const Primary: Story = {
```

Comment:
This includes all different modals in one story which doesn't really make sense anymore since they're now their separate stories, Storybook should always be a mirror to what we have in Figma!

I suggest this story obj getting deleted